getting_started_tutorials/cuml_sklearn_colab_demo.ipynb

```diff
@@ -0,0 +1,4310 @@
+{
```

Comment:
<p>Could be nice to add a "reminder" here that we're only doing these restarts for the sake of the demo. In practice, you'd just load the accelerator once up front and you're all set.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai-community/showcase/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>